build.xml

```diff
@@ -481,6 +496,7 @@
     <!--
         The build target builds all the .class files
     -->
+    <target name="_build_subprojects" depends="_build-accord" description="Builds all subprojects needed for the build" />
```

Comment:
Hopefully we can add the dtest-api to this soon :)

Reply:
yep, this is why I went this route, to make life easier later on... been trying to make most of this generic with jvm-dtest in mind